pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/ColumnMetadata.java

```diff
@@ -36,6 +35,7 @@
 @InterfaceAudience.Private
 @SuppressWarnings("rawtypes")
 public interface ColumnMetadata {
+  int INDEX_NOT_FOUND = -1;
```

Comment:
What is the contract around this constant? Seems it is used as the index size when index doesn't exist?

Reply:
Added javadoc.